bin/README.md

```diff
@@ -180,6 +188,25 @@ export images="angular-on-nginx django solr schema-spy"
 # The routes for the project
 export routes="angular-on-nginx django solr schema-spy"
 ```
+## Setting up pull secrets
+To get around docker rate limiting, you may need to create an account on docker.io or use artifactory. In order to do this, you will need to set up
+a pull secret. Here are the environment variables you should add to the bottom of your settings.sh or settings.local.sh.  
+- `USE_PULL_CREDS` is a boolean flag that allows to specify whether or not you want to build the pull credentials.   
+- `PULL_CREDS` will be the name of the newly created pull credentials.  
```

Comment:
We could probably just use one environment variable with the name of the pull credentials: if set, we can assume pull credentials need to be created (less combinations of flags that can be set).

Reply:
now env vars are populated with default values and can be overriden